components/buttons/loginButton.js

```diff
@@ -0,0 +1,47 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import styled from "styled-components";
+import Image from "next/image";
+import GitHubIcon from "../../public/github_logo.png"
+
+
+
+
+export default function LoginButton() {
+
+
+  const { data: session } = useSession();
+ 
+ 
+
+  return (
+    <>
+      <Button onClick={() => signIn("github")}>
+          <Image
+            src={GitHubIcon}
+            alt="GitHub Icon"
+           width={25}
+            objectFit="contain"
```

Comment:
I'm curious, doesn't the image component need both a `width` and `height` attribute?

Reply:
if you use obectFit="contain" it atomatically readjusts the missing property :)

Reply:
🦊